src/content/changelog/4-61-0-2020-04-06.md

```diff
@@ -8,16 +8,16 @@ changelog:
 
 ### Added
 
-- You can now add tags when creating an LKE cluster ([POST /lke/clusters](/api/v4/lke-clusters/#post)). While the `tags` property existed before, the implementation logic has now been added to support adding tags to an LKE cluster.
+- You can now add tags when creating an LKE cluster ([POST /lke/clusters](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-create)). While the `tags` property existed before, the implementation logic has now been added to support adding tags to an LKE cluster.
 
-- A `tags` property was added to the Update Kubernetes Cluster ([PUT /lke/clusters/{clusterId}](/api/v4/lke-clusters-cluster-id/#put)) endpoint. You can now update or remove existing tags on your LKE clusters. The `tags` property accepts an array of tags to be applied to your LKE cluster for organizational purposes.
+- A `tags` property was added to the Update Kubernetes Cluster ([PUT /lke/clusters/{clusterId}](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-update)) endpoint. You can now update or remove existing tags on your LKE clusters. The `tags` property accepts an array of tags to be applied to your LKE cluster for organizational purposes.
 
-- The [Images collection](/api/v4/images-image-id) now contains the `updated` property. This property returns the date that the current Image was last updated.
+- The [Images collection](https://www.linode.com/docs/api/images/#image-view) now contains the `updated` property. This property returns the date that the current Image was last updated.
 
 ### Changed
 
-- The [StackScript collection's](/api/v4/linode-stackscripts-stackscript-id) `deployments_total` property is now filterable. See [Filtering and Sorting](/api/v4/#filtering-and-sorting) to learn how to search API v4 collections.
+- The [StackScript collection's](https://www.linode.com/docs/api/stackscripts/#stackscript-view) `deployments_total` property is now filterable. See [Filtering and Sorting](/https://www.linode.com/docs/api/) to learn how to search API v4 collections.
```

Comment:
same here: http://www.linode.com/docs/api/#filtering-and-sorting

Reply:
pushed!